src/Microsoft.Azure.NotificationHubs/Microsoft.Azure.NotificationHubs.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
Can probably also support .NET Standard 2.1

Reply:
2.1 and 2.0 are backwards compatible and we don't have any code that requires 2.1 APIs so I don't see any value in adding 2.1 into the mix. Replacing 2.0 with 2.1 however will break anyone running .NET Core 2.0 apps.